ONSdigital/dp-frontend-renderer
# Release/1.10.0

### What

- Added Table of Contents link to the Previous Versions section of a dataset page.
- Updated a link to return to the current version of the dataset you are on, rather than the most recent version.

### How to review

- Navigate to a dataset page for an older version and check to see that the table of contents link is in the Previous Versions section.
- Check to see that clicking on the 'refer to dataset' link takes you to the dataset page for the version you are viewing.

### Who can review

Anyone
